core/qt-ble.cpp

```diff
@@ -195,6 +195,9 @@ void BLEObject::addService(const QBluetoothUuid &newService)
 
 	report_info("Found service %s", to_str(newService).c_str());
 
+	auto service = controller->createServiceObject(newService, this);
+	all_services.append(service);
```

Comment:
I think it is preferrable that we exclude bluetooth chip firmware upgrade services that are filtered below with `is_known_bad_service()` - we don't want to accidentally mess with those.